tests/component/test_task.py

```diff
@@ -26,3 +40,61 @@ def test___tasks_with_different_names___hash___not_equal(generate_task):
     task2 = generate_task("MyTask2")
 
     assert hash(task1) != hash(task2)
+
+
+@pytest.mark.device_name("bridgeTester")
+@pytest.mark.parametrize(
+    "shunt_resistor_value, shunt_resistor_location, shunt_resistor_select, shunt_resistor_source, bridge_resistance, skip_unsupported_channels",
+    [
+        (100000, 12465, 0, 0, 0.2, True),
```

Comment:
Please add another test case for each function that uses the default arguments. This would also catch the missing `.value` for the enum parameters.